web/src/containers/Summary/_Summary.scss

```diff
@@ -160,7 +160,68 @@ $trade-tab--arrow-size: 0.4rem;
 		opacity: 0.5;
 	}
 }
-
+.ant-select-single:not(.ant-select-customize-input) .ant-select-selector {
+	background-color: #202020;
+	color: #a8a8b1;
+}
+.ant-select-focused:not(.ant-select-disabled).ant-select-single:not(.ant-select-customize-input)
+	.ant-select-selector,
+.ant-select-item-option-active:not(.ant-select-item-option-disabled) {
+	border-color: #d9d9d9;
+	background-color: #212121;
+	color: #a8a8b1;
+}
+.ant-select-dropdown,
+.ant-select-item-option-selected:not(.ant-select-item-option-disabled) {
+	background-color: #212121;
+	color: #a8a8b1;
+}
+.ant-select-item {
+	color: #a8a8b1;
+}
+.ant-select-dropdown {
+	border: 1px solid #a8a8b1;
+}
+.digital-market-wrapper {
+	display: flex;
+	justify-content: center;
+	padding-top: 5rem;
+	.market-wrapper {
+		width: 70%;
+		.dropdown-container {
+			margin-bottom: 2rem;
+		}
+		.gray-text {
+			color: #b2b3b3;
+		}
+		.header-container {
+			padding-bottom: 1rem;
+			border-bottom: 2px solid #bbbbbc;
```

Comment:
We should not hard-code colors. We already have different shades of border colors